doc/poster/example1_hdf5.cpp

```diff
@@ -0,0 +1,57 @@
+#include <vector>
+#include "hdf5.h"
+
+void data_io() {
+  hid_t  file_id, dset_id, dspace_id, group_id; /* identifiers */
+  herr_t status;
+
+  // ==== Writing a simple vector to HDF5
+
+  // Setting up dataset dimensions and input data
+  int ndims = 1;
+  hsize_t dims[ndims];
+  dims[0]      = 50;
+  std::vector<double> data(50, 1);
+
+  // Opening a file
+  file_id = H5Fopen("/tmp/new_file.h5", H5F_ACC_RDWR, H5P_DEFAULT);
```

Comment:
```suggestion
  file_id = H5Fcreate("/tmp/new_file.h5", H5F_ACC_TRUNC, H5P_DEFAULT);
```
(or don't call it "new" file :))